druid-shell/src/platform/windows/window.rs

```diff
@@ -1679,7 +1679,9 @@ impl Cursor {
         let name = match self {
             Cursor::Arrow => IDC_ARROW,
             Cursor::IBeam => IDC_IBEAM,
+            Cursor::Pointer => IDC_HAND,
             Cursor::Crosshair => IDC_CROSS,
+            // not correct, maybe polyfill this
```

Comment:
yea, can we open an issue here? Not sure what the best solution is, I'd probably prefer to deprecate the open hand from this enum. 😢